### PR TITLE
add method for get user mention

### PR DESCRIPTION
**Pull request by sheldygg**
# Description

I'm tired of writing like this
```python
html.link(message.from_user.full_name, f'tg://user?id={message.from_user.id}')
```
so i did it pull request

## Type of change

- adding method to get to get mention of user

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have added tests that prove my fix is effective or that my feature works as expected
- [x] New and existing unit tests pass locally with my changes
- [x] My changes generate no new warnings or errors
- [x] My changes are compatible with minimum requirements of the project
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: tests/test_api/test_types/test_user.py [hunk 1]
```diff
@@ -14,7 +14,7 @@ class TestUser:
             ["User", " ", "User  "],
             [" ", " ", "   "],
         ],
-    )
-    def test_full_name(self, first: str, last: str, result: bool):
+    )        
+    def test_get_mention_html(self, first: str, last: str, result: bool):
         user = User(id=42, is_bot=False, first_name=first, last_name=last)
-        assert user.full_name == result
+        assert user.mention_html()
```
Comment (evgfilim1) [inline tests/test_api/test_types/test_user.py:20]: Don't edit tests, add new ones instead

Comment (evgfilim1) [inline tests/test_api/test_types/test_user.py:20]: Don't assert string values unless you're checking that something is expected to be non-empty. In your case, you must check if `mention_html` returns exact string. Then try to add arguments and see if it works properly. And don't forget to test `mention_markdown`.

**Review by evgfilim1 — CHANGES_REQUESTED** [review]
Please fix your code formatting by running `black` and `isort` from project directory.

**Review by sheldygg — COMMENTED** [review]
fix

File: aiogram/types/user.py [hunk 2]
```diff
@@ -40,3 +41,17 @@ def full_name(self) -> str:
         if self.last_name:
             return f"{self.first_name} {self.last_name}"
         return self.first_name
+
+    @property
+    def url(self) -> str:
+        return f"tg://user?id={self.id}"
```
Comment (JrooTJunior) [inline aiogram/types/user.py:47]: Is preffered to use `create_tg_link` function which gives an identical result
```python
create_tg_link("user", id=self.id)
```
Can be imported from:
```python
from aiogram.utils.link import create_tg_link
```

**Review by JrooTJunior — CHANGES_REQUESTED** [review]

File: aiogram/types/user.py [hunk 3]
```diff
@@ -40,3 +41,17 @@ def full_name(self) -> str:
         if self.last_name:
             return f"{self.first_name} {self.last_name}"
         return self.first_name
+
+    @property
+    def url(self) -> str:
+        return create_tg_link("user", self.id)
```
Comment (JrooTJunior) [inline aiogram/types/user.py:47]:
```suggestion
        return create_tg_link("user", id=self.id)
```

**Review by JrooTJunior — CHANGES_REQUESTED** [review]

**Review by JrooTJunior — APPROVED** [review]

**Review by evgfilim1 — APPROVED** [review]
Looks good, thanks

**Pull request merged**